toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -276,16 +321,38 @@ func (im *IsoMaker) prepareWorkDirectory() {
 // copyStaticIsoRootFiles copies architecture-independent files from the
 // Mariner repo directories.
 func (im *IsoMaker) copyStaticIsoRootFiles() {
-	staticIsoRootFilesPath := filepath.Join(im.resourcesDirPath, "assets/isomaker/iso_root_static_files/*")
 
-	logger.Log.Debugf("Copying static ISO root files from '%s'.", staticIsoRootFilesPath)
+	if im.resourcesDirPath == "" && im.grubCfgPath == "" {
+		logger.Log.Panicf("Missing required parameters. Must specify either the resources directory or provide a grub.cfg.")
+	}
+
+	if im.resourcesDirPath != "" {
```

Comment:
Does it make sense for both `resourcesDirPath` and `grubCfgPath` to be specified considering that the only resource file is the `grub.cfg` file?

Reply:
Well, the `resources` folder contains both arch-specific files and non-arch-specific files.
For the non-arch-specific files, grub.cfg is indeed the only file there today. But from the way the code is written, there's an assumption that this non-arch-specific group could include other files and the code will not need to change should a new file is dropped there.
Given that we have only grub.cfg, we can remove unused functionality... I'm thinking of leaving since it will be useful if new files are added to that rresources directory...